lib/console.js

```diff
@@ -3,12 +3,9 @@
 const util = require('util');
 
 function Console(stdout, stderr) {
-  if (!(this instanceof Console)) {
-    return new Console(stdout, stderr);
-  }
-  if (!stdout || typeof stdout.write !== 'function') {
-    throw new TypeError('Console expects a writable stream instance');
-  }
+  if (!(this instanceof Console)) return new Console(stdout, stderr);
```

Comment:
We have an ESLint setup. But, in general, you should change the bare minimum in your PRs. Otherwise, it makes tools like git blame harder to use.